layman/overlays/source.py

```diff
@@ -20,6 +20,11 @@
 import sys
 import shutil
 import subprocess
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
```

Comment:
You added configparser import, but I don't see it used in the remaining patch.

Doe sit get used in a later patch?  I haven't looked yet.  If so, it should be in the patch that uses it.